docs/language-clients.md

```diff
@@ -55,6 +55,16 @@ Client is expected to always launch a single instance of the server and check fo
 It is assumed that paths to these folders will be provided as part of `workspaceFolders`
 in the `initialize` request per LSP.
 
+## Code Actions
+
+The server implements a set of opt-in code actions which perform different actions for the user. The code action request is sent from the client to the server to compute commands for a given text document and range. These commands are typically code fixes to either fix problems or to beautify/refactor code.
+
+### Format Document
+
+The server will format a given document or text range according to Terraform formatting conventions.
+
+This can be enabled for a single language by specifying `source.formatAll.terraform-ls` or for all languages by specifying `source.formatAll`. Some editors like VS Code enable further customization of when code actions are performed.
+
```

Comment:
Not sure how much detail we want here since configuring this is very client specific